### PR TITLE
Update customizing-your-repositorys-social-media-preview.md

### LINKED ISSUE #27309: Project "Social Preview" tip says the opposite of the interface
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

`content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md`

content:

> Tip: Your image should be a PNG, JPG, or GIF file under 1 MB in size. For the best quality rendering, we recommend keeping the image at 640 by 320 pixels

### What part(s) of the article would you like to see updated?

> Tip: Your image should be a PNG, JPG, or GIF file under 1 MB in size. For the best quality rendering, we recommend keeping the image at 640 by 320 pixels

https://github.com/github/docs/blob/cc03f596ca5cf0632717a363adb0a51e84e6c7d2/content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md?plain=1#L25C1-L26C1

The interface and screenshot below say the opposite.

### Additional information

I'd be interested to know which is correct.

### PR DESCRIPTION
**Pull request by lariscy**
### Why:

Closes: #27309

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Suggested edit from issue #27309

### Check off the following:

- [X] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@lariscy Thanks so much for submitting a PR! I'll get this triaged for review :zap:

File: content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Until you add an image, repository links expand to show basic information about
 
 {% tip %}
 
-**Tip:** Your image should be a PNG, JPG, or GIF file under 1 MB in size. For the best quality rendering, we recommend keeping the image at 640 by 320 pixels.
+**Tip:** Your image should be a PNG, JPG, or GIF file under 1 MB in size. For the best quality rendering, we recommend keeping the image at least 640 by 320 pixels (1280 by 640 pixels for best display).
```
Comment (mchammer01) [inline content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md:25]:
```suggestion
**Tip:** Your image should be a PNG, JPG, or GIF file under 1 MB in size. For the best quality rendering, we recommend a size of at least 640 by 320 pixels (1280 by 640 pixels for best display).
```

**Review by mchammer01 — DISMISSED** [review]
LGTM, thanks for fixing this inaccuracy in the docs ✨ 
I've made a minor suggestion that I am going to apply.

**Review by mchammer01 — APPROVED** [review]
:shipit:

**Pull request merged**